config/environments/production.rb

```diff
@@ -33,6 +33,17 @@
     protocol: "https"
   }
   config.action_mailer.asset_host = "https://#{production_hostname}"
+
+  config.action_mailer.smtp_settings = {
+    address: ENV.fetch("SMTP_HOSTNAME"),
+    port: ENV.fetch("SMTP_PORT", 587), 
+    enable_starttls_auto: true,
+    user_name: ENV.fetch("SMTP_USERNAME"),
+    password: ENV.fetch("SMTP_PASSWORD"),
```

Comment:
Do these env vars need to be put in `example.env` ?

Reply:
"need" might be a strong word because the app will totally work in dev and test without them but I think we should add them to `example.env` as a heads up for the dev/ops person

Reply:
@eoinkelly maybe commented out? They're not required in any environment other than prod

Reply:
That seems like the right compromise